crates/common/tedge_config/src/tedge_config_cli/tedge_config_dto.rs

```diff
@@ -81,6 +81,14 @@ pub(crate) struct CumulocityConfigDto {
     #[doku(example = "your-tenant.cumulocity.com", as = "String")]
     pub(crate) url: Option<ConnectUrl>,
 
+    /// HTTP Endpoint for the Cumulocity tenant, with optional port.
+    #[doku(example = "http.your-tenant.cumulocity.com:1234", as = "String")]
+    pub(crate) http: Option<HostPort<443>>,
+
+    /// MQTT Endpoint for the Cumulocity tenant, with optional port.
+    #[doku(example = "mqtt.your-tenant.cumulocity.com:1234", as = "String")]
+    pub(crate) mqtt: Option<HostPort<8883>>,
```

Comment:
I would define somewhere 2 constants for the default HTTP and MQTT ports.
```suggestion
    /// HTTP Endpoint for the Cumulocity tenant, with optional port.
    #[doku(example = "http.your-tenant.cumulocity.com:1234", as = "String")]
    pub(crate) http: Option<HostPort<HTTP>>,

    /// MQTT Endpoint for the Cumulocity tenant, with optional port.
    #[doku(example = "mqtt.your-tenant.cumulocity.com:1234", as = "String")]
    pub(crate) mqtt: Option<HostPort<MQTT>>,
```

Reply:
I agree, simple and sufficient to reduce the "magic number" code smell